bitnami/aspnet-core/README.md

```diff
@@ -412,6 +418,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 4.0.0
+
+This major release includes the network policies and the component labels needed to apply the network policies to our pods.
```

Comment:
Explain that, to upgrade the release, they may need to delete the release and re-deploy